docs/pod_network_shaping.md

```diff
@@ -0,0 +1,20 @@
+### Pod Network shaping
+Scenario to introduce network latency, packet loss, and bandwidth restriction in the Pod's network interface. The purpose of this scenario is to observe faults caused by random variations in the network.
+
+##### Sample scenario config for egress traffic shaping (using plugin)
+'''
+- id: pod_egress_shaping
```

Comment:
thoughts on adding this - id: pod_egress_shaping as part of pod_network_outage config?

Reply:
Yes, that sounds good too.